locales/en-US/editor.properties

```diff
@@ -125,7 +125,7 @@ CMD_TOGGLE_QUICK_DOCS=Quick Docs
 
 # Drag and Drop
 
-DND_MAX_FILE_SIZE_EXCEEDED=file exceeds maximum supported size: 3MB
+DND_MAX_SIZE_EXCEEDED=file exceeds maximum supported size: {0} MB.
```

Comment:
@gideonthomas can you clarify what we need to do here, now that this string is in Thimble?  Do we do both?